framework/src/test/java/com/retailmenot/scaffold/webelement/AbstractWebElementTests.java

```diff
@@ -138,6 +142,20 @@ public void testExistsFalse() {
         assertFalse("The element should not exist", testAbstractWebElement.exists());
     }
 
+
+    /**
+     * Test the mock log error created from {@link MockLogs}
+     */
+    @Test
+    public void testGetErrorLog() {
+        LogEntries logEntries = mockWebDriver.manage().logs().get(LogType.BROWSER);
```

Comment:
I suggest following more the standard mock pattern where you set the expected return value for the mock as part of your test. If MockLogs class changes, it will break your test.

Reply:
Made some updates to accommodate for the setting of the data during the test data setup